### ISSUE TITLE
Data Preview stuck after selecting json for Object detection scenario

### ISSUE THREAD

**Issue by JeremyMahieu** [dotnet/machinelearning-modelbuilder#2460]
**System Information (please complete the following information):**
 - Model Builder Version 16.14.3.2307502
 - Visual Studio Version 17.4.4

**Describe the bug**
When selecting an input json file with image labeling, datapreview loads endlessly, can't go to the next step.

**To Reproduce**
1. Add a Machine Learning Model to an empty console project.
2. Start an "Object detection" scenario
3. Set up a workspace.
4. Go to https://ml.azure.com/ and do data labeling on a bunch of images, export the .json
5. Select the json in the Add data step

**Expected behavior**
Select the .json and be able to go to the next step. Or get an error saying the json is not correct.

**Screenshots**
![image](https://user-images.githubusercontent.com/23409814/215254043-2faea658-6f3b-4b0e-a9f6-e8e1424a37c6.png)
![image](https://user-images.githubusercontent.com/23409814/215254067-6b092b7e-e315-42cf-83de-363c0a7daeb0.png)

```json
{
  "images": [
    {
      "id": 1,
      "width": 4032,
      "height": 3024,
      "file_name": "UI/2023-01-28_061527_UTC/asdf/20221213_145249.jpg",
      "coco_url": "AmlDatastore://redacted/UI/2023-01-28_061527_UTC/asdf/20221213_145249.jpg",
      "absolute_url": "https://redacted.blob.core.windows.net/azureml-blobstore-redacted/UI/2023-01-28_061527_UTC/asdf/20221213_145249.jpg",
      "date_captured": "2023-01-28T06:18:25.732655Z"
    }
  ],
  "annotations": [
    {
      "id": 1,
      "category_id": 1,
      "image_id": 1,
      "area": 0.002,
      "bbox": [
        0.27528852681602173,
        0.6812627291242362,
        0.04752206381534285,
        0.033604887983706755
      ]
    }
  ],
  "categories": [
    {
      "id": 1,
      "name": "redacted"
    }
  ]
}
```

**Comment by beccamc** [comment]
@JeremyMahieu Sorry you ran into this problem. You used the Azure ML Studio -> Data labelling?  Unfortunately we currently only support files generated by [Vott](https://learn.microsoft.com/en-us/dotnet/machine-learning/tutorials/object-detection-model-builder#create-a-new-vott-project). 

@luisquintanilla Thoughts on adding this as a data source for OD?

**Comment by luisquintanilla** [comment]
Makes sense to support the Azure ML format as input. Let's toss it to the backlog for now.

@JakeRadMSFT can correct me here but I believe that's what we do already (though not exposed as an option on input).

**Comment by JeremyMahieu** [comment]
No I didn't use Vott. Vott has been unmaintained for more than a year, I thought the json files would be compatible. And since the training is on azure etc.
At the very least there should be some error saying there's a problem, not just infinite loading.
Thanks for the work.

**Comment by Bjornej** [comment]
Are there any plans to support any formats other than VOTT? The tool has been discontinued and support for other formats could simplify the training ....
